integration-tests/src/dca.rs

```diff
@@ -201,6 +202,99 @@ mod omnipool {
 		});
 	}
 
+	#[test]
+	fn buy_schedule_execution_should_emit_swapped_events() {
+		TestNet::reset();
+		Hydra::execute_with(|| {
+			//Arrange
+			init_omnipool_with_oracle_for_block_10();
+
+			let dca_budget = 1000 * UNITS;
+			let amount_out = 100 * UNITS;
+			let schedule_id = 0;
+			let schedule1 = schedule_fake_with_buy_order(PoolType::Omnipool, HDX, DAI, amount_out, dca_budget);
+			create_schedule(ALICE, schedule1);
+
+			//Act
+			run_to_block(11, 12);
+
+			//Assert
+			let swapped_events = get_last_swapped_events();
+			let last_two_swapped_events = &get_last_swapped_events()[swapped_events.len() - 2..];
+			pretty_assertions::assert_eq!(
+				last_two_swapped_events,
+				vec![
+					RuntimeEvent::AmmSupport(pallet_amm_support::Event::Swapped {
+						swapper: ALICE.into(),
+						filler: Omnipool::protocol_account(),
+						filler_type: pallet_amm_support::types::Filler::Omnipool,
+						operation: pallet_amm_support::types::TradeOperation::ExactOut,
+						inputs: vec![Asset::new(HDX, 140421094367051)],
+						outputs: vec![Asset::new(LRNA, 70210545436437)],
+						fees: vec![Fee::new(LRNA, 35105272718, Omnipool::protocol_account()),],
+						operation_id: vec![
```

Comment:
I am inclined to rename this to something in line of `operation_stack` does it ring more bells?

Reply:
I like this, lemme rename it!